app/forms/update_user_profile_form.rb

```diff
@@ -4,7 +4,7 @@ class UpdateUserProfileForm
   include FormMobileValidator
   include FormPasswordValidator
 
-  attr_accessor :mobile, :email, :current_password, :password, :totp_enabled
+  attr_accessor :user, :mobile, :email, :current_password, :password, :totp_enabled
```

Comment:
Why this change, and the change from `@user` to `user`? It doesn't seem to be related to this PR. Also, we don't want the user to be writeable from the outside. It should only be readable. If what's bothering you is the reference to `@user` instead of `user`, that can solved by adding `attr_reader :user` under the `private` section.